src/mon/MonMap.cc

```diff
@@ -374,7 +374,7 @@ void MonMap::_add_ambiguous_addr(const string& name,
 	return;
       }
       if (!contains(addr)) {
-	add(name, entity_addrvec_t(addr));
+	add(name, entity_addrvec_t(addr), priority);
```

Comment:
can we remove the default parameter of `priority=0` of `MonMap::add(const string &name, const entity_addrvec_t &addrv, int priority)`?